test/net.test.js

```diff
@@ -4,454 +4,247 @@ var assert = require('assert');
-        });
-
-        test('Taxable income is gt 18m and lt 32m, dependent 2', () => {
-          let netSalary = net(32_402_236, 2);
-
-          assert.equal(netSalary.toNumber(), 28_330_001.098);
-        });
-
-        test('Taxable income is gt 18m and lt 32m, dependent 3', () => {
-          let netSalary = net(32_402_236, 3);
-
-          assert.equal(netSalary.toNumber(), 28_760_001.159);
-        });
-
-        test('Taxable income is gt 18m and lt 32m, dependent 4', () => {
-          let netSalary = net(32_402_236, 4);
-
-          assert.equal(netSalary.toNumber(), 28_980_001.159);
-        });
-        
-        test('Taxable income is gt 18m and lt 32m, dependent 5', () => {
-          let netSalary = net(32_402_236, 5);
-
-          assert.equal(netSalary.toNumber(), 29_000_001.22);
-        });
-        //_____________________________________
-
-        test('Taxable income is eq 32m, dependent 1', () => {
-          let netSalary = net(47_884_187, 1);
-
-          assert.equal(netSalary.toNumber(), 39_918_276.104);
-        });
-
-        test('Taxable income is eq 32m, dependent 2', () => {
-          let netSalary = net(47_884_187, 2);
-
-          assert.equal(netSalary.toNumber(), 40_798_276.104);
-        });
-
-        test('Taxable income is eq 32m, dependent 8', () => {
-          let netSalary = net(47_884_187, 8);
-
-          assert.equal(netSalary.toNumber(), 43_985_345.13);
-        });
-        //______________________________________
-
-
-
-        test('Taxable income is gt 32m and lt 52m, dependent 1', () => {
-          let netSalary = net(60_900_000, 1);
-
-          assert.equal(netSalary.toNumber(), 49_753_250);
-        });
-
-        test('Taxable income is gt 32m and lt 52m, dependent 11', () => {
-          let netSalary = net(60_900_000, 11);
-
-          assert.equal(netSalary.toNumber(), 56_871_000);
-        });
-
-        //_________________________________________________
-
-        test('Taxable income is gt 52m and lt 80m, dependent 1, region 1', () => {
-          let netSalary = net(68000000, 1, 1);
-
-          assert.equal(netSalary.toNumber(), 55_025_000);
-        });
-
-        
-        test('Gross is eq 93.6m and taxable income lt 80m (test BHTN), dependent 1, region 1', () => {
-          let netSalary = net(93_600_000, 1, 1);
-
-          assert.equal(netSalary.toNumber(), 72_940_800);
-        });
-
+        describe('Before 1/7/2023', function () {
+
+            test('Gross Income <= 29.8 million (20 times of gross basic)', function () {
+                //Gross Income <= 29.8 million:
+                // Social Insurance (SI): 8% Gross
+                // Health Insurance (HI): 1.5% Gross
+                // Unemployment Insurance (UI): 1% Gross
+
+                const netSalary = net(10, 0, 1, new Date("2023-06-01"));
+                assert.equal(netSalary, 8.95);
+            })
+
+            test('29.8 million < Gross Income < 88.4 million', function () {
```

Comment:
pls Make the test case more Descriptive (Tips: What is the **OBJECTIVE** of that test case?)